public/keymaps/b/basekeys_slice_rev1_default.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard": "basekeys/slice/rev1",
+  "keymap": "default_4704aba",
+  "commit": "4704aba55cac8ee6d382498aafac9e61e48bfadf",
```

The commit hash, and thus the keymap suffix, are derived from the last commit to modify the board's default keymap on qmk_firmware's `master` branch:

```
noroadsleft ~/qmk_firmware (master)
$ git log -1 --pretty=format:"%H  %s by %an" -- keyboards/basekeys/slice/keymaps/default/keymap.c
de58b0765954ed8fc9bb800f389808fb7eee6eda  Slice Keyboard (#8464) by 2Moons-JP
```

I use this to track when the default keymaps on this repo need to be updated.

```suggestion
  "keymap": "default_de58b07",
  "commit": "de58b0765954ed8fc9bb800f389808fb7eee6eda",
```